temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/components/DateTimeRow.kt

```diff
@@ -40,22 +42,24 @@ fun DateTimeRow(
             iconStart = R.drawable.ic_date
         ) {
             ivyContext.datePicker(
-                initialDate = dateTime.convertUTCtoLocal().toLocalDate()
+                initialDate = with(timeConverter) { dateTime.toLocalDate() }
```

Comment:
Move the date-time picking to the VM, see `EditTransactionViewModel` and how it does it using the new datetime picker